pkg/csource/csource_test.go

```diff
@@ -207,15 +207,15 @@ csource6(&AUTO)
 `,
 			output: fmt.Sprintf(`
 NONFAILING(memcpy((void*)0x%x, "\x12\x34\x56\x78", 4));
-syscall(SYS_csource2, 0x%xul);
+syscall(SYS_csource2, /*a=*/0x%xul);
```

Comment:
Let's adjust `sys/test/csource.txt` to make these tests more interesting. Right now they all just print a single single-char paramer name.

Reply:
Done ;)